setup/src/Magento/Setup/Model/ConfigOptionsList/DriverOptions.php

```diff
@@ -0,0 +1,38 @@
+<?php
+namespace Magento\Setup\Model\ConfigOptionsList;
+
+use Magento\Framework\Config\ConfigOptionsListConstants;
```

Comment:
Due to [Magento technical guidelines](https://devdocs.magento.com/guides/v2.2/coding-standards/technical-guidelines.html) we should use strict type declaration  (`declare(strict_types=1)`) for all new files.